wiki/osu!wiki_contribution_guide/en.md

```diff
@@ -898,7 +914,7 @@ All linking must be done manually, and must be done using strict static linking
 
 A few things to take note when linking:
 
-- **Any typos in the link** will cause **Error 404 (content not found; link to nowhere because of nonexistent file)**.
+- **Any typos in the link** will cause a**404 Not found** error.
 - **No section anchors**.
```

Comment:
remove this
section anchors/links now work (they just follow strict rules just like the links)